lib/pure/htmlgen.nim

```diff
@@ -808,3 +808,102 @@ runnableExamples:
       annotation(encoding = "application/x-tex", title = "Latex on Web", r"x^{2} + y")
     )
   ) == """<math><semantics><annotation encoding="application/x-tex" title="Latex on Web">x^{2} + y</annotation></semantics></math>"""
+
+
+runnableExamples:
```

Comment:
Come on, that's not an "example", that's a unit test and nobody likes to read it.